test/library/standard/IO/readBinary/multiloc/binaryBytes.chpl

```diff
@@ -4,7 +4,7 @@ on Locales[0] {
     var reader = openReader("./input.bin");
 
     on Locales[1] {
-        var b: bytes = createBytesWithNewBuffer(b"");
+        var b: bytes = bytes.createCopyingBuffer(c"");
```

Comment:
1. Did we not deprecate `NewBuffer` overloads with the same-type arguments completely? i.e. `bytes.createCopyingBuffer(x: bytes)` shouldn't exist.
2. `b""` is a `bytes` literal, `c""` is a `c_string` literal. Was it because 1, you weren't able to pass `b""`?
3. this line should just be `var b = b""`;

Reply:
Yes exactly, `b""` fails because `bytes.createCopyingBuffer(x: bytes)` doesn't exist.

Out of curiosity, I tried `var b = b""` (and `var s = ""` for the string test). Both tests pass with that.

Reply:
> Out of curiosity, I tried var b = b"" (and var s = "" for the string test). Both tests pass with that.

That's the way to go for these tests.